test/util/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "util"
```

Comment:
Can you call this `test-util` so that its purpose is a bit clearer? (`util` has the potential to clash with the various `util.rs` modules around.)